sequencer.c

```diff
@@ -3162,6 +3162,26 @@ int setup_reflog_action(struct replay_opts *opts, const char *commit,
 	return 0;
 }
 
+int checkout_onto(struct replay_opts *opts,
+		  const char *onto_name, const char *onto,
+		  const char *orig_head, unsigned verbose)
+{
+	struct object_id oid;
+	const char *action = reflog_message(opts, "start", "checkout %s", onto_name);
+
+	if (get_oid(orig_head, &oid))
+		/* TODO: translation? */
+		die("%s: not a valid SHA1", orig_head);
```

Comment:
Where does this error message comes from? At least it needs s/SHA1/OID/.

Reply:
It comes from here: https://github.com/git/git/blob/master/builtin/update-ref.c#L414-L415

update-ref was originally called by checkout_onto().

Reply:
Ok, I still think "OID" or "object id" are better than SHA1 these days.

Reply:
OK. Do I wrap this in a `_()`?

Reply:
Yes, please!